### PR TITLE
Force markdown rendering on all files regardless of file's extension. 

### LINKED ISSUE #28: Add Markdown Rendering
There might be a prompt kit module or setting to enable this.

Removing line numbers is a + too.

### PR DESCRIPTION
**Pull request by ntdkhiem**

### CODE REVIEW

**Comment by BoraxTheClean** [comment]
Let's wait until #32 is merged and then make sure this still works with that change. #32 seems to have broken markdown rendering atm.

**Review by BoraxTheClean — APPROVED** [review]
This is working for me. Does it look good for you locally too?

**Comment by ntdkhiem** [comment]
Awesome

**Pull request merged**
